tests/test_idpi/test_lat_ops.py

```diff
@@ -12,14 +14,12 @@
 from idpi.operators.hzerocl import fhzerocl
 
 
-@pytest.fixture
-def data_dir():
```

Comment:
The purpose of this fixture was to override the top-level data_dir fixture that the fieldextra fixture was using to define the default paths. With this change, you need to pass in updated paths to the fieldextra call.